src/components/Auth/SignUpForm.schema.js

```diff
@@ -0,0 +1,21 @@
+import Joi from '@hapi/joi';
+import { createValidationResolver } from '../form';
+
+const schema = Joi.object({
```

Comment:
Nice! So if we wanted to validate a create resource form for example, we'd create a CreateResource.schema.js file as well.

Reply:
Yup, that's the convention I personally prefer so we can keep both schema and form alongside instead grouping all the schema somewhere else together.

Reply:
Thanks, sounds good :)